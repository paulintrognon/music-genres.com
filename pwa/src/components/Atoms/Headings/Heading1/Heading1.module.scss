.container {
  text-transform: uppercase;
  font-size: var(--font-size-32);
  font-family: 'Montserrat Subrayada';
}
