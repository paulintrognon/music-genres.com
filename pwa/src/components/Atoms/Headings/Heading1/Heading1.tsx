import classnames from 'classnames'
import styles from './Heading1.module.scss'

type Props = React.ComponentProps<'span'>
const Heading1: React.FC<Props> = ({ className, children }) => {
  return <span className={classnames(styles.container, className)}>{children}</span>
}
export default Heading1
