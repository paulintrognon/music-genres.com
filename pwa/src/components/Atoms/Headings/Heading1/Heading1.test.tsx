import React from 'react'
import renderer from 'react-test-renderer'
import Heading1 from './Heading1'

it('should render the Heading1', () => {
  expect(
    renderer.create(<Heading1 className="testClass">Super Title</Heading1>).toJSON()
  ).toMatchSnapshot()
})
