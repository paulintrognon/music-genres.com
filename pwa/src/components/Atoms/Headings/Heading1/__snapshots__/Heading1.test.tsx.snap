// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Heading1 1`] = `
<span
  className="container testClass"
>
  Super Title
</span>
`;
